B1/Index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Dự Án 1</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<center>
+		<div class="contentTop">
+			<div id="t_left">
```

Comment:
Reducing using id if we don't have any specific purpose. Please using class instead.
Check the whole files.